cmd/bridge/main.go

```diff
@@ -158,6 +159,12 @@ func main() {
 		offClusterAlertManagerURL.Path = "/api"
 	}
 
+	offClusterMeteringURL := &url.URL{}
+	if *fK8sModeOffClusterMetering != "" && *fK8sMode == "off-cluster" {
```

Comment:
in https://github.com/openshift/console/pull/2527 we did ` && *fK8sMode == "off-cluster"` for each off cluster arg:
```
if *fK8sModeOffClusterMetering != "" && *fK8sMode == "off-cluster" {
```
I think it's nicer to move from the code from L149 to L166 into the `case "off-cluster":` in L316 and create `offClusterAlertManagerURL`, `offClusterPrometheusURL` and `offClusterMeteringURL` inside the case, WDYT ?